### DISCUSSION TITLE
Provide Full Width Option

### DISCUSSION THREAD

**Discussion by cekageka** [GitbookIO/community#450] [Feature Requests]
It's nice to have a "full width" option for tables and maps (neither of which are _really_ wide), but it would be so much better if you supported a _real_ full width option for content. More and more people are using 27" and larger screens, and when reading documents it doesn't make sense to only use a narrow strip in the middle of the screen that has to be scrolled horizontally.

<img width="2560" alt="Screenshot 2024-01-12 at 18 07 21" src="https://github.com/GitbookIO/community/assets/1116271/7505308f-8826-4ddc-8fe8-80920c9a7fae">

**Comment by MichaMican** [comment]
Also for big tabels and images this would be a gamechanger

**Comment by crnnr** [comment]
I have also recently come across this problem! This would be very useful

**Comment by enpatrik** [comment]
Looks like they've actually released "wide" page option now: https://gitbook.com/docs/changelog#new-wide-page-width-option 👀 

Would've been nice if they had dropped a comment here for all the issue subscribers ;-)

It solved my use case anyway!
